File: Java/GovernoSingleton.java
package teste;

public class GovernoSingleton {
	/* Atributos e metodos*/
	
	public String proximaEleicao = "2022";
	public String nomeDoPais = "Brasil";
	
	public void realizarEleicao() {
		System.out.println("Realizando eleição!");
	}
	
	/* Estrutura Singleton */
	private static GovernoSingleton instancia;
	private GovernoSingleton(){}
	public static synchronized GovernoSingleton pegarInstancia() {
		if(instancia == null) {
			instancia = new GovernoSingleton();
		}
		return instancia;
	}
}

File: PHP/GovernoSingleton.php
<?php

class GovernoSingleton{
  /* Campos */

  public $nomeDoPais = "Brasil";
  public $proximaEleicao = "2022";

  public function realizarEleicao(){
      echo "Realizando...<br>";
  }

  /* Estrutura */  
  private static $instancia = null;  
  private function __construct(){}
  private function __clone(){} 
  public static function pegarInstancia(){
      if(self::$instancia == null){
          self::$instancia = new GovernoSingleton();
      }
      return self::$instancia;
  }
}